portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -373,46 +373,38 @@ definition {
 		property test.name.skip.portal.instance = "XSSUsecase#ViewURLValidate";
 
 		var maliciousURL1 = '''
-				http://localhost:8080/c/portal/flash?title=abc</title><script%0d%0a>alert(3.1415926)</script%0d%0a><title>aaa
```

Comment:
Hey @csierra, could you have someone from your team look into this? 

Since we no longer expose the URL, there's no possibility of attack anymore. I tested manually and that URL will now redirect to home with no XSS.

Just double-checking in case you'd rather keep it here.